docs/blog/newsletter-july-2019.rst

```diff
@@ -0,0 +1,115 @@
+
+------------------------------------------
+For those who might consider freelancing …
+------------------------------------------
+
+… a generous Slack contributor provided a wealth of information and advice about what to look for and what to expect in the land of freelance work.
+
+We were planning to write this up for the newsletter, but very kindly the contributor did the work for us! So you can read the write-up here: `Real Talk: Freelance/Contract Writing <https://dacharycarey.com/2019/06/12/real-talk-freelance-contract-writing/>`__
+
+A few other people also weighed in with some comments. Some don't work with a 1099 but set up their own corporations (S-Corp or LLC), and then work as their own employees. This approach can provide tax benefits for US workers, and can sometimes make you more appealing to a larger employer. People also suggested that professional support, in the form of a lawyer or accountant, is important if you're planning to work freelance long-term.
+
+And if you're going the independent route, don't forget to consider staffing agencies. They get a cut of whatever the client pays, but they sometimes provide benefits depending on the contract, and can ease the paperwork burden.
+
+------------------------------
+What to do about text wrapping
+------------------------------
+
+A very specific topic, but one that comes up for almost everyone working with docs-as-code: should you mandate text wrapping in your docs source, and if so, how? For some background before digging into this summary, check out `To Wrap or Not to Wrap: One of Life's Greatest Questions <https://about.gitlab.com/2016/10/11/wrapping-text/>`_ 
+
+Forcing a hard wrap at a particular character count or column helps avoid `reflow problems <https://www.w3.org/TR/WCAG21/#reflow>`_, but causes other problems. For example, hard wraps:
+
+* create awkward line breaks on devices with different screen sizes
+* wreak havoc on diffs
+* make text more difficult to parse for people who are translating to different languages and people with cognitive disabilities
+
+Hard wraps also interfere with using filters like `grep <http://man7.org/linux/man-pages/man1/grep.1.html>`_ to search for words and phrases within a file. Soft wrap is an option for preserving searchability, but it requires everyone to use a text editor with soft wrap support.
+
+Several people mentioned `one sentence per line <https://rhodesmill.org/brandon/2012/one-sentence-per-line/>`_ as a solution, noting that it's the `recommended approach for AsciiDoc <https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/asciidoc-recommended-practices.adoc#one-sentence-per-line>`_.
+
+-----------------------------------------------------------------
+From our sponsor: Spotlight on The Not-Boring Tech Writer podcast 
+-----------------------------------------------------------------
+
+Excitingly, this month's newsletter is sponsored by KnowledgeOwl!
```

Comment:
This is really cool, but I'm not quite sure how to introduce it. I feel like it deserves something to say "this is new" though.

Reply:
Are we going to be having regular newsletter sponsors from now on? In that case, maybe it would help to say something in the introduction that this is a thing now. That would make this smoother?

Reply:
Agreed. We should also link to the prospectus (http://www.writethedocs.org/newsletter/sponsorship/) as a way of selling another one :)

Reply:
Yeah, let me have another go at this bit